src/index.js

```diff
@@ -0,0 +1,29 @@
+import './style.css';
+import {
+  getStorageData, clearCompletedList, clearCheckBox, addTodo, saveEdit, displayToDo, getIsEditing,
+} from './module/addRemove.js';
+
+window.onload = () => {
+  getStorageData();
+  displayToDo();
+};
+
+const refreshBtn = document.querySelector('.refresh-btn');
+refreshBtn.addEventListener('click', () => {
+  clearCheckBox();
+  window.location.reload();
+});
+
+const desc = document.querySelector('.taskList');
+desc.addEventListener('keyup', (event) => {
+  if (event.keyCode === 13) {
+    event.preventDefault();
+    if (!getIsEditing())addTodo();
+    else saveEdit();
+  }
```

Comment:
Please use brackets in the conditional to give a better understanding of the code:

```
if (condition) {
// do something
}
else{
// do something
}
```